### DISCUSSION TITLE
[Bug]: Detected CSRF Attack! Do not do evil things with pimcore ... ;-)

### DISCUSSION THREAD

**Discussion by abdelrhmanyosry** [pimcore/pimcore#17330] [Q&A / Support] [answered]
### Pimcore version

11

### Steps to reproduce

just when trying to login in pimcore 

### Actual Behavior

redirect me this page!
[image](https://github.com/pimcore/pimcore/assets/121692656/eeb62c22-511a-4e37-ae33-76da99d4f638)

### Expected Behavior

should open the admin panel

**Comment by kingjia90** [comment]
Any more info on how to reproduce it? 
It seems working fine on my end and on the public demos https://demo.pimcore.com/

Could you please retry there again? maybe with a different browser?

From past "investigations", i recall there could be some autocomplete plugin that alters the CSRF token, also double-click could also trigger it.

**Comment by dpfaffenbauer** [comment, marked as answer]
be careful with the latest 1Password Update, it automatically clicks the login button for you, if you click again, you do a double submit resulting in that error
